go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/ONSdigital/dp-frontend-feedback-controller
+
+go 1.14
```

Comment:
This should be `1.13` to match the docker image in `ci/build.yml` (note that we don't need the patch version so it's `1.13` not `1.13.15`)
To change this, don't manually change this file, run the following commands instead…
```
go mod edit -go=1.13
go mod tidy
```
Don't be alarmed if these commands change more than just this one line.